README.md

```diff
@@ -262,7 +262,87 @@ export default NuxtAuthHandler({
 })
 ```
 
-Note that there's way more options inside the `nextAuth.options` object, see [here](https://next-auth.js.org/configuration/options#options) for all available options.
+Note: The above credential-provider example is taken over in part from the [NextAuth.js credentials example](https://next-auth.js.org/configuration/providers/credentials). It is _not_ considered safe for production usage and you would need to adapt it further, e.g., by calling another service that provides authentication, such as [strapi](#example-with-a-custom-strapi-jwt-provider)
+
+##### Example with a custom Strapi JWT provider 
+
+This section gives an example of how the `NuxtAuthHandler` can be configured to use Strapi JWTs for authentication via the `CredentialsProvider` provider.
+
+Ther are five places to configure `nuxt-auth` to work with Strapi:
+- `STRAPI_BASE_URL` in `.env`: Add the Strapi environment variable to your .env file.
+- [`runtimeConfig.STRAPI_BASE_URL`-key in `nuxt.config.ts`](#nuxtconfigts): Add the Strapi base url env variable. 
+- [`runtimeConfig.STRAPI_API_TOKEN_SECRET`-key in `nuxt.config.ts`](#nuxtconfigts): Optional: Allows server-to-server/machine-to-machine/headless usage through Strapi API Tokens. 
```

Comment:
let's remove everything related to `STRAPI_API_TOKEN_SECRET` -> as you say it's optional and not actually required for the `nuxt-auth` setup. Please remove:
- [ ] this line,
- [ ] the two lines below that also reference this